web/yo/app/scripts/controllers/gene.js

```diff
@@ -2885,6 +2891,7 @@ angular.module('oncokbApp')
                     // This will only happen if the currentReviewer is not empty
                     $scope.fileEditable = false;
                 }
+                $scope.geneEditable = $scope.fileEditable;
```

Comment:
What you said will happen only in some cases. 
When file.editable is false, the watch on fileEditable is not going to be triggered. But at this time, the condition check in displayCheck() will be executed, and geneEditable will be assigned to be false. So I removed it